### DISCUSSION TITLE
winget user vs system context in task sequence SCCM

### DISCUSSION THREAD

**Discussion by Arnaud3131** [Romanitho/Winget-Install#117] [Q&A]
first of all a big thank you to Romanitho for his great work!

winget-install works very well during a SCCM sequence task, it correctly installs the applications that I ask it to and updates those that need to be updated, however I have a problem once the session is opened for a standard user if I do a winget in cmd, it does not recognize it as if it was not installed...

in the user's locappdata there is no more winget, however if I go to "C:\Program Files\WindowsApps" I have Microsoft.AppInstallerxxx but I cannot use it.

I would like to understand why users cannot use it? and how to fix it?

moreover during the winget-install.ps1 script, 99% of the time it fails to update winget itself...

**Comment by Romanitho** [comment]
Hi, thanks.

Do you block the microsoft store ?

**Reply by Arnaud3131** [reply]
Hi,

Effectively the MS Store is block by GPO

**Reply by Romanitho** [reply]
so maybe winget is not installed/maintained.

**Reply by Arnaud3131** [reply]
ok i understand, thanks
